CMakeLists.txt

```diff
@@ -41,6 +41,11 @@ endif()
 # Include GNUInstallDirs to get canonical paths
 include(GNUInstallDirs)
 
+# Declare the option for code coverage
+option(SIMBODY_COVERAGE
+       "Adding ability to assess test coverage. Code coverage is possible with gcc or clang."
```

Comment:
I think this line is over 80 columns! Consider: "Adding ability to assess test coverage (requires gcc or clang)".